source/patterns/@aws-solutions-constructs/aws-lambda-elasticsearch-kibana/README.md

```diff
@@ -25,6 +25,13 @@
 ## Overview
 This AWS Solutions Construct implements the AWS Lambda function and Amazon Elasticsearch Service with the least privileged permissions.
 
+**Some cluster configurations (e.g VPC access) require the existence of the `AWSServiceRoleForAmazonElasticsearchService` Service-Linked Role.**
+
+**You will need to create the service-linked role using the AWS CLI:**
```

Comment:
...using the AWS CLI once in any account using this construct (it may have already been run to support other stacks).**